### PR TITLE
[libepoxy] Update to 1.5.5

### PR DESCRIPTION
**Pull request by ras0219**
This change improves upon the situation in #12275.

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

File: ports/libepoxy/portfile.cmake [hunk 1]
```diff
@@ -1,76 +1,30 @@
...
-    )
-    file(COPY ${OUT_PATH_RELEASE}/include DESTINATION ${CURRENT_PACKAGES_DIR})
-    file(COPY ${OUT_PATH_RELEASE}/lib DESTINATION ${CURRENT_PACKAGES_DIR})
-    file(RENAME ${CURRENT_PACKAGES_DIR}/lib ${CURRENT_PACKAGES_DIR}/bin)
+    set(OPTIONS -Denable-glx=yes -Denable-egl=yes -Denable-x11=yes -Dc_std=gnu99)
 endif()
 
+vcpkg_configure_meson(
+    SOURCE_PATH ${SOURCE_PATH}
+    OPTIONS ${OPTIONS}
```
Comment (Neumann-A) [inline ports/libepoxy/portfile.cmake:23]: option tests should be deactivated.

File: ports/libepoxy/portfile.cmake [hunk 2]
```diff
@@ -1,76 +1,30 @@
...
-    message(STATUS "Installing ${TARGET_TRIPLET}")
-    vcpkg_execute_required_process(
-        COMMAND make install
-        WORKING_DIRECTORY ${SOURCE_PATH}
-        LOGNAME install-${TARGET_TRIPLET}-release
-    )
-    file(COPY ${OUT_PATH_RELEASE}/include DESTINATION ${CURRENT_PACKAGES_DIR})
-    file(COPY ${OUT_PATH_RELEASE}/lib DESTINATION ${CURRENT_PACKAGES_DIR})
-    file(RENAME ${CURRENT_PACKAGES_DIR}/lib ${CURRENT_PACKAGES_DIR}/bin)
+    set(OPTIONS -Denable-glx=yes -Denable-egl=yes -Denable-x11=yes -Dc_std=gnu99)
```
Comment (Neumann-A) [inline ports/libepoxy/portfile.cmake:18]:
```suggestion
    set(OPTIONS -Dglx=yes -Degl=yes -Dx11=true -Dc_std=gnu99)
```

File: ports/libepoxy/portfile.cmake [hunk 3]
```diff
@@ -1,76 +1,30 @@
...
 )
 
 if (VCPKG_TARGET_IS_WINDOWS OR VCPKG_TARGET_IS_OSX)
-    vcpkg_configure_meson(SOURCE_PATH ${SOURCE_PATH}
-        OPTIONS
-            -Denable-glx=no
-            -Denable-egl=no)
-    vcpkg_install_meson()
-    vcpkg_copy_pdbs()
+    set(OPTIONS -Denable-glx=no -Denable-egl=no -Denable-x11=no -Dc_std=c99)
```
Comment (Neumann-A) [inline ports/libepoxy/portfile.cmake:16]:
```suggestion
    set(OPTIONS -Dglx=no -Degl=no -Dx11=false -Dc_std=c99)
```

Comment (Neumann-A) [reply ports/libepoxy/portfile.cmake:16]: options are defined in `meson_options.txt` seems like the names are wrong here.

File: ports/libepoxy/portfile.cmake [hunk 4]
```diff
@@ -1,76 +1,30 @@
 vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
```
Comment (Neumann-A) [inline ports/libepoxy/portfile.cmake:1]: Is this really true or does this only apply to windows?

Comment (ras0219) [reply ports/libepoxy/portfile.cmake:1]: I don't know, to be honest.

**Pull request merged**